core-web/libs/sdk/analytics/README.md

```diff
@@ -0,0 +1,93 @@
+# @dotcms/analytics
+
+`@dotcms/analytics` is the official dotCMS JavaScript library for Content Analytics that helps track events and analytics in your webapps. Currently available as an IIFE (Immediately Invoked Function Expression) module for direct browser usage.
+
+## Features
+
+-   **Simple Browser Integration**: Easy to implement via script tags using IIFE implementation
+-   **Event Tracking**: Simple API to track custom events with additional properties
+-   **Automatic PageView**: Option to automatically track page views
+-   **Debug Mode**: Optional debug logging for development
+
+## Installation
+
+Include the script in your HTML page:
+
+```html
+<script src="analytics.iife.js"></script>
+```
+
+## Configuration
+
+The script can be configured using data attributes:
+
+-   **data-analytics-server**: URL of the server where events will be sent. If not provided, it defaults to the current location (window.location.href).
+-   **data-analytics-debug**: Presence of this attribute enables debug logging (no value needed)
+-   **data-analytics-auto-page-view**: Presence of this attribute enables automatic page view tracking (no value needed)
+-   **data-analytics-key**: Required. API key for authentication with the analytics server. This key is provided by the DotCMS Analytics app.
+
+## Usage
+
+### Automatic PageView Tracking
+
+When `data-analytics-auto-page-view` is enabled, the library will automatically send a page view event to dotCMS when the page loads. If this attribute is not present, you'll need to manually track page views and other events using the tracking API.
+
+```html
+<!-- Automatic page view tracking enabled & debug logging enabled -->
+<script
+    src="ca.min.js"
+    data-analytics-server="http://localhost:8080"
+    data-analytics-key="dev-key-123"
+    data-analytics-auto-page-view
+    data-analytics-debug></script>
+
+<!-- Without automatic tracking - events must be sent manually -->
+<script
+    src="ca.min.js"
+    data-analytics-server="http://localhost:8080"
+    data-analytics-debug
+    data-analytics-key="dev-key-123"></script>
+```
+
+## Roadmap
+
+The following features are planned for future releases:
+
+1. **Manual Event Tracking**
+
+    - Manual track events support for IIFE implementation
+
+2. **Headless Support**
+
+    - React integration for event tracking
+    - Next.js integration for event tracking
+
+3. **Angular Support**
+    - Angular integration for event tracking
```

Comment:
I would assume that Angular is inside the headless, maybe we can change Headless Support for React or NextJS Support

Reply:
Yes, it is. Changed